pytest_mergify/__init__.py

```diff
@@ -40,50 +40,49 @@ def pytest_terminal_summary(
 
         terminalreporter.section("Mergify CI")
 
-        if self.mergify_tracer.tracer_provider is None:
-            if not self.mergify_tracer.token:
-                terminalreporter.write_line(
-                    "No token configured for Mergify; test results will not be uploaded",
-                    yellow=True,
-                )
-                return
-
-            if not self.mergify_tracer.repo_name:
-                terminalreporter.write_line(
-                    "Unable to determine repository name; test results will not be uploaded",
-                    red=True,
-                )
-                return
-
+        if not self.mergify_ci.token:
             terminalreporter.write_line(
-                "Mergify Tracer didn't start for unexpected reason (Please contact Mergify support); test results will not be uploaded",
-                red=True,
+                "No token configured for Mergify; test results will not be uploaded",
+                yellow=True,
             )
             return
 
-        try:
-            self.mergify_tracer.tracer_provider.force_flush()
-        except Exception as e:
+        if not self.mergify_ci.repo_name:
             terminalreporter.write_line(
-                f"Error while exporting traces: {e}",
+                "Unable to determine repository name; test results will not be uploaded",
                 red=True,
             )
-        else:
-            terminalreporter.write_line(
-                f"MERGIFY_TEST_RUN_ID={self.mergify_tracer.test_run_id}",
-            )
+            return
 
-        try:
-            self.mergify_tracer.tracer_provider.shutdown()
-        except Exception as e:
+        if self.mergify_ci.tracer_provider is None:
             terminalreporter.write_line(
-                f"Error while shutting down the tracer: {e}",
+                "Mergify Tracer didn't start for unexpected reason (please contact Mergify support); test results will not be uploaded",
                 red=True,
             )
+        else:
```

Comment:
nitpic, we can early return and reduce indent
```suggestion
            return
```